index.html

```diff
@@ -44,16 +44,142 @@ <h2 class="car">I'm a software developer</h2>
                 <a href="#"><img class="angelist-logo" alt="angellist-logo" src="img/angellist.svg" /></a>
             </li>
         </ul>
+
+        <br><br>
+        
+        <div class="projects">
```

Comment:
- [ ] Kindly use a `<section>` element to wrap this entire section as this is best practice for HTML5 semantics. Please Note that the "My recent works" heading is also part of the section as it is the section title.